src/imu/mpu9255/mpu9255.cpp

```diff
@@ -0,0 +1,156 @@
+#include "mpu9255.hpp"
+
+namespace imu {
+
+auto Mpu9255::Init(void) noexcept -> types::DriverStatus {
+  CreateSensorPointer();
+
+  SetInitConfigMPU9255();
+
+  if (InitAllSensors()) {
+    SetToInitialized();
+    return types::DriverStatus::OK;
+  }
+  return types::DriverStatus::HAL_ERROR;
+}
+
+auto Mpu9255::CreateSensorPointer(void) noexcept -> void {
+  if (gyroscope_ == NULL) {
```

Comment:
for the null value of a pointer there is nullptr and is slightly different from NULL

Reply:
Adapted.